src/modules/orders/entities/buyer-order.entity.ts

```diff
@@ -0,0 +1,63 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  JoinColumn,
+  ManyToMany,
+  JoinTable,
+  Column,
+} from 'typeorm';
+
+import { Order } from 'src/modules/orders/entities/order.entity';
+import { User } from 'src/modules/users/user.entity';
+
+@Entity('buyer_order')
+export class BuyerOrder {
+  @ApiProperty({
+    example: '61c674384-f944-401b-949b-b76e8793bdc9',
+    description: 'The ID of the buyer`s order',
+    type: String,
+  })
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @ApiProperty({
+    example: 15000,
+    description: 'The price of the buyer`s order',
+    type: Number,
+  })
+  @Column({ nullable: false, type: 'decimal', precision: 10, scale: 2 })
+  price: number;
+
+  @ApiProperty({
+    example: '61c674384-f944-401b-949b-b76e8793bdc9',
+    description: 'The ID of the user',
+    type: String,
+  })
+  @ManyToOne(() => User, (user) => user.buyerOrders, { onDelete: 'CASCADE' })
+  @JoinColumn({ name: 'user_id' })
+  user: User;
+
```

Comment:
Also we should have here fields for: 1) shipping price; 2) is the payed (boolean)? 3) paymentId (to link with stripe).

Reply:
1) done 2) I am not shour about it, as we already have field "status" that shows if the order is paid 3) is it possible to link with stripe using "id" field of buyer_order?

Reply:
2) But that "status" field is in the order entity, not here in the buyer_order. A user pays for a buyer_order, and the buyer_order status changes to "paid." This means that our shop received money in Stripe. However, the status of a specific order depends on the actions of its vendor and the Canadian Post.  3) Upon payment, we receive a payment ID from Stripe in the form of pm_1Phqc5Cj7lDMnopKDgRwhwAb. We need a field to store this ID so that the site admin can communicate with Stripe and vendors and refer to this ID. It is also advisable to have a field with the payment date to make it easier to search for payments and to sort all orders by this field.

Reply:
done